src/main/java/org/skriptlang/skript/addon/AddonModule.java

```diff
@@ -44,4 +52,43 @@ default boolean canLoad(SkriptAddon addon) {
 		return true;
 	}
 
+	/**
+	 * Returns the documentation category which this module belongs to.
+	 *
+	 * @return The category of this module.
+	 */
+	default @NotNull Set<Category> category() {
```

Comment:
I would prefer for this to be separated out in another interface. Maybe something like `Categorizable`.